### ISSUE TITLE
Update man page with latest CLI commands

### ISSUE THREAD

**Issue by zeroXten** [pki-io/docs#2]
Perhaps wait until the CLI UX is established.

**Comment by thanasisk** [comment]
should we not revisit this topic?

**Comment by zeroXten** [comment]
Yeah, but as part of the refactor the CLI commands a changing a little bit, so not much point doing this until that is finished.
